src/lib/Behat/BusinessContext/ContentViewContext.php

```diff
@@ -42,15 +42,19 @@ public function startCreatingArticle(string $name): void
     }
 
     /**
+     * @given I start editing the content
      * @Given I start editing the content in :language language
      */
-    public function startEditingContent(string $language): void
+    public function startEditingContent(string $language = null): void
     {
         $rightMenu = new RightMenu($this->utilityContext);
         $rightMenu->clickButton('Edit');
 
         $languagePicker = new LanguagePicker($this->utilityContext);
-        $languagePicker->chooseLanguage($language);
+
+        if ($languagePicker->isVisible()) {
+            $languagePicker->chooseLanguage($language);
+        }
```

Comment:
Have you checked, how it's going to behave if there'll be a bug and language picker appears when it shouldn't? 
`chooseLanguage` function will throw an error, or will choose first available language? If it's second case we probably wont detect such bug.

Reply:
Good question, I can add an additional assertion here, something like:
if a picker is visible then check if there is more than one language to pick from, what do you think?

Reply:
I think that is a good solution.